protocol/synthetix/contracts/interfaces/IRewardsManagerModule.sol

```diff
@@ -5,23 +5,46 @@ pragma solidity ^0.8.0;
  * @title Module for connecting rewards distributors to vaults.
  */
 interface IRewardsManagerModule {
+    /**
+     * @notice Emitted when the pool owner or an existing reward distributor sets up rewards for vault participants.
+     * @param poolId The id of the pool on which rewards were distributed.
+     * @param collateralType The collateral type of the pool on which rewards were distributed.
+     * @param distributor The reward distributor associated to the rewards that were distributed.
+     * @param amount The amount of rewards that were distributed, denominated with 18 decimals of precision.
```

Comment:
```suggestion
     * @param amount The amount of rewards that were distributed.
```

I think the amount is agnostic to decimal precision. It just gets passed through to the payout function.